README.md

```diff
@@ -1,54 +1,56 @@
 # Vamp Forklift command line client
 
-Vamp Forklift is a command line client written in golang and allows to easily set up Organizations and Environments in Vamp.
```

Comment:
Some of this actually still applies, doesn't it?

Reply:
I will update it